docs/SyncVsAsync.md

```diff
@@ -0,0 +1,93 @@
+---
+id: sync-vs-async
+title: Sync mode vs async
+---
+
+## Async mode
+
+Every browser/element function call returns a Promise and needs to be awaited to get result.
+
+It's not allowed to chain functions.
```

Comment:
This statement seems kind of vague, maybe adding why is not allowed might help with understanding.

Reply:
I'll just remove it